Formula/ripgrep.rb

```diff
@@ -19,6 +19,14 @@ def install
 
     bin.install "target/release/rg"
     man1.install "doc/rg.1"
+
+    # Completion scripts are generated in the crate's build directory, which
+    # includes a fingerprint hash. Try to locate it first
+    out_dir = Dir["target/release/build/ripgrep-*/out"].first
+    if out_dir
```

Comment:
This test is a new addition. `out_dir` detection *should* never fail, but in case someday it does, let's not break with a cryptic error message.